README.md

```diff
@@ -50,6 +50,7 @@ When a PR passes the above check, `jira-description-action` will also add the is
 
 | key                    | description                                                                                                                                                                                                                                                                                                     
 | ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `jira-issue-key`         | The JIRA issue key. If key is not found the value is an empty string |****
```

Comment:
you also need to add it here, otherwise it won't work
https://github.com/cakeinpanic/jira-description-action/blob/master/action.yml